src/core_ocean/shared/mpas_ocn_tracer_hmix_redi.F

```diff
@@ -205,6 +207,9 @@ subroutine ocn_tracer_hmix_Redi_tend(meshPool, scratchPool, layerThickness, laye
                iCellSelf = 2
             endif
 
+            areaSum = areaSum + 0.5_RKIND * dvEdge(iEdge) * dcEdge(iEdge)
+      !      rediKappaCell(iCell) = rediKappaCell(iCell) + 0.5_RKIND*RediKappa(iEdge) * invAreaCell * dvEdge(iEdge) * dcEdge(iEdge)
```

Comment:
@vanroekel, is this for debugging, and should it be removed?

Reply:
yes, thanks for catching that I will remove.